### ISSUE TITLE
Cannot open audio results with keyboard navigation on all results grid

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#1242]
## Description

It is not possible to open the audio results with keyboard on all results grid.

## Reproduction

1.  Go to https://search.openverse.engineering/search?q=birds
2.  Tab to the first audio result
3.  Hit enter

## Expected Results
The link should open the audio results

## Actual result
The audio plays

## Resolution

- [x] 🙋 I would be interested in resolving this bug.

**Comment by zackkrida** [comment]
Oh no! This _was_ working previously—thanks for noticing it.
